gp-ant-task/example/build.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<project name="gp-ant-task-1.1.5-SNAPSHOT" default="download-translations" 
+	xmlns:gp="antlib:com.ibm.g11n.pipeline.ant">
+	
+	<taskdef  uri="antlib:com.ibm.g11n.pipeline.ant" resource="com/ibm/g11n/pipeline/ant/antlib.xml">
+		<!-- If the gp-ant-task jar is not added to ANT_HOME/lib folder, 
+			then you may include the classpath for the jar like in the comment below-->
+		<!--<classpath path="lib/gp-ant-task-1.1.5-SNAPSHOT-with-dependencies.jar"/>-->
```

Comment:
I think it's not a good idea to put a specific version, especially with SNAPSHOT.
I propose to change path value to use gp-ant-task-x.x.x.jar. A user reading the comment should be able to understand x.x.x is a version.